libs/payments/ui/src/lib/client/components/MetricsWrapper.tsx

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import * as Sentry from '@sentry/nextjs';
+import { WithContextCart } from '@fxa/payments/cart/cart.types';
+import { useEffect } from 'react';
+
+export function MetricsWrapper({
+  cart,
+  children,
+}: {
+  cart?: WithContextCart;
+  children: React.ReactNode;
+}) {
+  useEffect(() => {
+    cart &&
+      Sentry.getGlobalScope().setTag('metricsOptedOut', cart.metricsOptedOut);
+  }, [cart]);
+
+  return <div>{children}</div>;
```

Comment:
Are these `<div>`'s necessary?

Reply:
Probably causing the strange screenshot

Reply:
it absolutely was :) fixed
![Screenshot 2024-10-02 at 9 37 49 AM](https://github.com/user-attachments/assets/77842da9-7eb1-4cb3-b026-7742d7018cc5)